src/handlers/v1/markets/[chainId]/[marketId]/leverage/index.ts

```diff
@@ -100,6 +102,12 @@ export const v1GetLeverageQuotationRoute: Route<GetLeverageQuotationRouteParams>
         loans: [{ token: baseToken.wrapped, amount: quotation.input.amount }],
       });
       const borrowAmount = repays.at(0).amount;
+      if (new BigNumberJS(borrowBalance).plus(borrowAmount).lt(baseBorrowMin)) {
+        throw newHttpError(400, {
+          code: '400.6',
+          message: `total borrow balance is less than baseBorrowMin: ${baseBorrowMin}`,
```

Comment:
```suggestion
          message: `target borrow balance is less than baseBorrowMin: ${baseBorrowMin}`,
```
Since the current response uses `targetPosition` as the predicted result